entrypoint.sh

```diff
@@ -27,6 +28,7 @@ fi && \
 git add . && \
 git commit -m 'Deploy to GitHub pages' && \
 git push --force $REMOTE_REPO master:$REMOTE_BRANCH && \
+curl -XPOST -H"Authorization: token ${GITHUB_TOKEN}" https://api.github.com/repos/${GITHUB_REPOSITORY}/pages/builds && \ # https://git.io/fjsFk
```

Comment:
As this API is in the preview period, [it requires to specify a custom media type in the `Accept` header][1]:

```bash
-H"Accept: application/vnd.github.mister-fantastic-preview+json"
```

[1]: https://developer.github.com/v3/repos/pages/#request-a-page-build